modules/@apostrophecms/schema/index.js

```diff
@@ -1261,6 +1264,28 @@ module.exports = {
         }
       },
 
+      // If a field has a following property and a parent,
+      // hoist that property values to the parent,
+      // if they start with `<`.
+      hoistFollowingFieldsToParent(field, parent) {
+        if (!parent || !field.following) {
+          return;
+        }
+        const following = typeof field.following === 'string'
```

Comment:
Is this a thing? I'm pretty sure we don't support `following` as anything other than an array.

Reply:
But if there are counterexamples already in our code then that is how it is.

Reply:
`const following = Array.isArray(field.following) ? field.following : [ field.following ];` is how it's done prior the change (code from the editor mixin).